platforms/quorum/configuration/deploy-network.yaml

```diff
@@ -43,40 +43,49 @@
       org: "{{ item }}"
       kubernetes: "{{ item.k8s }}"
     loop: "{{ network['organizations'] }}"
-  
+
   # Setup Vault-Kubernetes accesses and Regcred for docker registry
-  - name: "Setup vault"   
-    include_role: 
+  - name: "Setup vault"
+    include_role:
       name: "{{ playbook_dir }}/../../shared/configuration/roles/setup/vault_kubernetes"
     vars:
       policy_type: "quorum"
       name: "{{ item.name | lower }}"
       component_ns: "{{ item.name | lower }}-quo"
-      component_name: "{{ item.name | lower }}-vaultk8s-job"      
+      component_name: "{{ item.name | lower }}-vaultk8s-job"
       component_auth: "quorum{{ item.name | lower }}"
       component_type: "organization"
       kubernetes: "{{ item.k8s }}"
       vault: "{{ item.vault }}"
-      gitops: "{{ item.gitops }}"      
+      gitops: "{{ item.gitops }}"
       reset_path: "platforms/quorum/configuration"
     loop: "{{ network['organizations'] }}"
 
   # Generate Ambassador certificate for nodes
-  - name: "Create ambassador certificates for Nodes" 
-    include_role: 
+  - name: "Create ambassador certificates for Nodes"
+    include_role:
       name: create/certificates/ambassador
     vars:
       root_subject: "{{ network.config.subject }}"
+      name: "{{ item.name }}"
+      node_name: "{{ item.name }}"
       cert_subject: "{{ network.config.subject | regex_replace(',', '/') }}"
       services: "{{ item.services }}"
-      organisation: "{{ item.name | lower }}"
+      organisation: "{{ item.name }}"
+      org_name: "{{ item.name }}"
```

Comment:
Please remove these extra vars.
2 ways to do so
1. Either straightaway use the var as {{ item.name }} (Not a good practice as the item var can be found inside the loop as well)
2. Create a `loop_var` and name it as org, and then you can access vars like `org_name` as simply {{ org.name }}